packages/headless/src/controllers/core/facets/range-facet/date-facet/date-range.ts

```diff
@@ -83,23 +87,15 @@ export function buildDateRange(config: DateRangeOptions): DateRangeRequest {
 function buildDate(rawDate: DateRangeInput, options: DateRangeOptions) {
   const {dateFormat} = options;
   if (isRelativeDate(rawDate)) {
+    validateRelativeDate(rawDate);
```

Comment:
The buildDateRange method is an entry point for all controllers and action, it is used be the validateManualRange method.